back/src/whombat/schemas/annotation_tasks.py

```diff
@@ -50,6 +52,12 @@ class AnnotationTask(BaseSchema):
     status_badges: list[AnnotationStatusBadge]
     """Status badges for the task."""
 
+    clip: Clip
```

Comment:
Since not all Task queries require Clip/ClipAnnotation data, consider creating a separate class (e.g., AnnotationTaskFull) that includes this information. This would allow us to fetch the full data only when necessary.